app/src/main/res/values/tv_colors.xml

```diff
@@ -11,7 +11,9 @@
     <color name="tv_blue50" tools:keep="@color/tv_blue50">@color/photonBlue50</color>
     <color name="tv_blue60" tools:keep="@color/tv_blue60">@color/photonBlue60</color>
     <color name="tv_ink" tools:keep="@color/tv_ink">#0c0c15</color>
-    <color name="translucent_tv_ink" tools:keep="@color/translucent_tv_ink">#cc0c0c15</color>
+    <color name="translucent_tv_ink" tools:keep="@color/translucent_tv_ink">#2a2a2e</color>
```

Comment:
So, since we are now using the photon colour palette, can I just delete the `tv_colors.xml` file and use all those colours directly? And then `colors.xml` will just be for any colours not already in the photon palette.